cg/apps/housekeeper/hk.py

```diff
@@ -243,6 +243,10 @@ def last_version(self, bundle: str) -> Version:
             .first()
         )
 
+    def get_all_non_archived_spring_files(self) -> List[File]:
```

Comment:
Very subjective, but `get_unarchived_spring_files` is a shorter name if you want an alternative

Reply:
I think we should use `unarchived` instead of `non_archived` as well throughout.

Reply:
hmm, counter argument: non archived to me means that it has not been archived. unarchived can be mistaken with retrieved, ie undoing the archiving.

Reply:
Then lets keep the name 👍  Explicit is always better